src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -935,6 +867,7 @@ namespace winrt::TerminalApp::implementation
                 settingsItem.Click({ this, &TerminalPage::_SettingsButtonOnClick });
                 newTabFlyout.Items().Append(settingsItem);
 
+                auto actionMap = _settings.ActionMap();
```

Comment:
This seems unused right now

Reply:
It is used in the line below this one, and further down on line 893